src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1632,6 +1632,16 @@ elementFormDefault="qualified">
         <xs:documentation><!-- _locID_text="PackageTargetFallback" _locComment="" -->Allows packages using alternative monikers to be referenced in this project, which include older (e.g. dnxcore50, dotnet5.x) and Portable Class Library names.</xs:documentation>
       </xs:annotation>
     </xs:element>
+    <xs:element name="AssetTargetFallback" substitutionGroup="msb:Property">
+      <xs:annotation>
+        <xs:documentation><!-- _locID_text="AssetTargetFallback" _locComment="" -->Can be set to one or more targetframeworks. When determining package compatibility, if the package is not determined to have compatible assets for the project based on the projects targetframework, compatibility will be rechecked using each targetframework from the AssetTargetFramework project of the hosting project.</xs:documentation>
+      </xs:annotation>
+    </xs:element>
+    <xs:element name=" Nullable" substitutionGroup="msb:Property">
```

Comment:
I think we could do fancier intellisense for the values as in https://github.com/microsoft/msbuild/issues/4391#issuecomment-545073077 if we created a new type but I don't think it's worth the effort.

Reply:
Yep. Keeping this one simple.